onecodex/models/sample.py

```diff
@@ -19,11 +19,43 @@ def __repr__(self):
 
     @classmethod
     def where(cls, *filters, **keyword_filters):
+        """Query and retrieve a set of samples.
+
+        Parameters
+        ----------
+        limit : `int`, optional
+            If set, retrieve a maximum of `limit` samples. Note a limit of 1000 samples is automatically
+            enforced for queries of all public samples (`public=True`)
+        organization : `bool`, optional
+            If True, search all samples within your organization (including your samples)
+        public : `bool`, optional
+            If True, search all public samples (limited to 1000 results)
+        tags : `list`, optional
+            A list of optional Tags to filter by. Tags should be `Tag` objects retrieved
+            with `ocx.Tags.get()` or `ocx.Tags.where()`
+        project : `Project`, optional
+            Filter by a Project
+        **keyword_filters : dict, optional
+            Pass any additional sample or metadata attribute to filter by that attribute. Metadata filtering
+            is *not* currently supported for
+
+        Returns
+        -------
+        A `SampleCollection` object with samples matching the query
+        """
         from onecodex.models.collection import SampleCollection
 
-        public = keyword_filters.get("public", False)
-        instances_route = "instances" if not public else "instances_public"
-        limit = keyword_filters.get("limit", None if not public else 1000)
+        public = keyword_filters.pop("public", False)
+        organization = keyword_filters.pop("organization", False)
+        instances_route = "instances"
+        if organization is True:
+            instances_route = "instances_organization"
+        if public is True:
+            instances_route = "instances_public"
+
+        # Set default filters
+        keyword_filters["_instances"] = instances_route
+        keyword_filters.setdefault("limit", None if not public else 1000)
```

Comment:
not using the negative would be a bit clearer: `1000 if public else None`. Or set `limit = None` at the top and set it to 1000 in the existing `if public is True:` above